src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Updated `System.Diagnostics.DiagnosticSource` package version to
+  `8.0.0-rc.1.23419.4`.
```

Comment:
why not rc.2? https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/8.0.0-rc.2.23479.6

Reply:
Updated.